.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E128,E811,W503,E203
+max-line-length = 100
```

Comment:
Please move this config into `pyproject.toml`

Reply:
Can't. Flake8 doesn't support pyproject.toml.

Reply:
What the shame! Do you think it's worth moving it to `.pre-commit-config.yaml` then?

Reply:
Hmm, the only side-effect of that is that my local flake8 hooks in vim now will complain about this stuff. Ah well.